src/components/UniversalNavbarExpanded/UniversalNavbarExpanded.module.scss

```diff
@@ -0,0 +1,375 @@
+@import '../Media/Media.scss';
+@import '../Variables.scss';
+
+$tablet-small-end: 670px;
+
+.flex {
+  display: flex;
+}
+
+.itemsCenter {
+  align-items: center;
+}
+
+.hamburger {
+  position: fixed;
+  top: 24px;
+  right: 24px;
+  z-index: 2;
+
+  @include for-laptop-and-up {
+    display: none;
+  }
+}
+
+.phoneAndTablet {
+  position: relative;
+  height: var(--Space-64);
+  background-color: white;
+  padding: var(--Space-24) var(--ContainerPadding--Phone);
+  display: none;
+  align-items: center;
+  justify-content: flex-end;
+  display: flex;
+  @include for-laptop-and-up {
+    display: none;
+  }
+  &.visible {
+    z-index: 3;
+  }
+  .hamburger {
+    position: absolute;
+  }
+}
+
+.laptopAndUp {
+  height: var(--Space-64);
+  background-color: white;
+
+  display: none;
+  @include for-laptop-and-up {
+    display: block;
+  }
+}
+
+.laptopAndUpContainer {
+  max-width: 1440px;
+  margin: 0 auto;
+  padding: var(--Space-24);
+  height: var(--Space-64);
+
+  display: flex;
+  justify-content: space-between;
+  @include for-laptop-only {
+    padding: var(--Space-24);
+  }
+  @include for-desktop-only {
+    padding: var(--Space-24) var(--Space-48);
+  }
+}
+
+.logo {
+  height: 16px;
+  padding-right: var(--Space-40);
+  @media (min-width: $tablet-range-start) and (max-width: $tablet-small-end) {
+    padding-right: var(--Space-24);
+  }
+}
+
+.phoneLogo {
+  margin-right: auto;
+}
+
+.phoneLogoFancy {
+  margin-right: auto;
+}
+
+.cta {
+  padding-left: 21px;
+  @media (min-width: $tablet-range-start) and (max-width: $tablet-small-end) {
+    padding-left: 0;
+  }
+}
+
+.textLink {
+  padding-right: var(--Space-24);
+  @media (min-width: $tablet-range-start) and (max-width: $tablet-small-end) {
+    &:nth-child(2) {
+      padding-right: var(--Space-16);
+    }
+    &:nth-child(3) {
+      padding-right: 0;
+    }
+  }
+  a {
+    color: var(--GraySecondary--translucent);
+    &:hover,
+    &:active,
+    &[aria-current='page'] {
+      color: inherit;
+    }
+  }
+}
+
+.searchIcon {
+  @include for-phone-and-tablet {
+    display: none;
+  }
+  @include for-laptop-and-up {
+    margin-right: 18px;
+    padding-top: 3px;
+    &:hover {
+      opacity: 0.78;
+    }
+  }
+}
+
+.accountIcon {
+  @include for-phone-and-tablet {
+    display: none;
+  }
+  @include for-laptop-and-up {
+    padding-top: 3px;
+    &:hover {
+      opacity: 0.78;
+    }
+  }
+}
+
+/**
+ * Big green mobile hamburger menu
+ */
+.mobileMenu {
+  position: fixed;
+  padding: 24px 24px 40px;
+  height: 100vh;
+  width: 100vw;
+  top: 0;
+  left: 0;
+  background-color: var(--BrandForest);
+  color: white;
+  z-index: 1;
+  overflow-y: scroll;
```

Comment:
generally `auto` works best and will only show scrollbar if we in fact have overflow

Reply:
will revise